pegasus/sites.v3/code.org/views/learn_code_studio.haml

```diff
@@ -12,6 +12,10 @@
   else
     launch_path = "/starwars"
   end
+  if request.GET['company']
+    company = URI.escape(request.GET['company'])
+    response.set_cookie('company', {value: company, domain: ".code.org", path: '/', expires: Time.now + (365 * 24 * 3600)})
```

Comment:
why is this not a "session cookie" which will expire when the browser window is closed?